server/docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+services:

```

Comment:
I think previously we discussed and decided not to use the docker. Do you remember?